next.config.js

```diff
@@ -1,33 +1,18 @@
-const path = require('path')
-const fs = require('fs')
-const {
-  getHashicorpPackages,
-} = require('@hashicorp/platform-nextjs-plugin/dist/get-hashicorp-packages')
-
 const withHashicorp = require('@hashicorp/platform-nextjs-plugin')
 const withSwingset = require('swingset')
 
-// Determine HC packages which are included in the monorepo to ensure we are transpiling all of our dependencies
-const workspaceRoot = path.join(process.cwd(), 'packages')
-let hashicorpPackagesInWorkspaces = []
-
-fs.readdirSync(workspaceRoot).forEach((dir) => {
-  hashicorpPackagesInWorkspaces = hashicorpPackagesInWorkspaces.concat(
-    getHashicorpPackages(path.join(workspaceRoot, dir))
-  )
-})
-
 module.exports = withHashicorp({
-  transpileModules: [
-    'swingset',
-    '@hashicorp/sentinel-embedded',
-    ...hashicorpPackagesInWorkspaces,
-  ],
+  transpileModules: ['swingset', '@hashicorp/sentinel-embedded'],
 })(
   withSwingset({
     componentsRoot: 'packages/*',
     docsRoot: 'docs/*',
   })({
+    eslint: {
+      // Warning: Dangerously allow production builds to successfully complete even if
+      // your project has ESLint errors.
+      ignoreDuringBuilds: true,
+    },
```

Comment:
Ticket to add this to our next plugin: https://app.asana.com/0/1100423001970639/1200795708532158/f